src/Arcus.EventGrid.Publishing/EventGridPublisher.cs

```diff
@@ -30,6 +29,7 @@ public class EventGridPublisher : IEventGridPublisher
         private readonly EventGridPublisherOptions _options;
 
         private static readonly JsonEventFormatter JsonEventFormatter = new JsonEventFormatter();
+        private static readonly HttpClient HttpClient = new HttpClient();
```

Comment:
Isn't it possible to inject the HttpClient instance via the constructor ?

Reply:
Could be, yeah, but we didn't do that yet. 😄 
Maybe that would make it better as we introduce HTTP message handlers that append the dependency ID automatically. 🤔 

But yeah, we could open an issue for that to inject the `IHttpClient`, but this is not always used in a HTTP context, so not sure if that service dependency is always available, then.

Reply:
We can investigate this later and not block this PR, it would be good if you could create an issue for this.

Reply:
Added #225 .